<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <param name="max_linear_speed" type="double" value="0.25" />
    <param name="max_angular_speed" type="double" value="0.3" />

    <node pkg="emotiv_ros" name="emotiv_server" type="command.py" output="screen"/>
    <node pkg="emotiv_ros" name="emotiv_gazebo_client" type="emotiv_gazebo_client" output="screen"/>

    <arg name="debug" default="false" />
    <arg name="pause" default="false" />
    <arg name="world" default="$(find emotiv_ros)/worlds/empty_world.world" />
    <include file="$(find gazebo_ros)/launch/empty_world.launch">
            <arg name="world_name" value="$(arg world)" />
            <arg name="debug" value="$(arg debug)" />
            <arg name="paused" value="$(arg pause)" />
            <arg name="use_sim_time" value="true" />
    </include>
</launch>